extensions/ql-vscode/src/config.ts

```diff
@@ -714,7 +714,15 @@ export function showQueriesPanel(): boolean {
 
 const DATA_EXTENSIONS = new Setting("dataExtensions", ROOT_SETTING);
 const LLM_GENERATION = new Setting("llmGeneration", DATA_EXTENSIONS);
+const DISABLE_AUTO_NAME_EXTENSION_PACK = new Setting(
```

Comment:
You want it opt-out instead of opt-in? Any specific reason?

Reply:
Especially since it ends up creating a directory with a file in.

Reply:
I think it's a much nicer experience to have the extension pack created automatically. I've kept the previous option in for more advanced users that want to decide where to place their extension pack. Perhaps we can decide whether to make this a choice the first time the user creates an extension pack by letting them select whether to do it automatically or manually. For now, I think having this flow is fine.